core/src/main/scala/kafka/server/KafkaRaftServer.scala

```diff
@@ -123,7 +123,9 @@ class KafkaRaftServer(
     raftManager.startup()
     controller.foreach(_.startup())
     broker.foreach(_.startup())
-    info(logLineForSystemTests)
+    if (broker.isEmpty) {
```

Comment:
hmm.  why would we not want to print the startup message when there is a broker?

Reply:
This handles the case where we are running with just the controller role.  If we always log it here it will get logged twice because `Kip500Broker` logs it.  I think we could remove it from there and always log it here if you prefer.

The issue was that the server won't start (and even fixing that would leave ducktape broken due to the regex match failing).  This fixes both issues.

Reply:
thanks for the explanation!